tests/vm_test.go

```diff
@@ -1404,7 +1404,10 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			By("Check if the VMI has been recreated")
 			Eventually(func() types.UID {
 				vmi, err = virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
+				Expect(err).ToNot(And(HaveOccurred(), Not(MatchError(errors.IsNotFound, "VMI was not found"))))
```

Comment:
`And` ?
Shouldn't this be `Or`?

Reply:
The way `Expect` is used here nested in `Eventually` also might not be correct.

Reply:
Good points, I am now passing the Gomega and asserting by it.